Update sold vehicle in boss action by date

mr-capy opened the pull request:
this will order sold vehicle by date in dec order so that an owner of vehicle easily get list of all vehicles that are recently sold

mr-capy commented:
by merging this if in rp server anyone owner of vehicle shop have to see the recent sold vehicle so he can give paycheck to his employees this will make much easy for him

mr-capy commented:
why failed?

iSentrie commented:
> why failed?

i guess u can ignore that, but u should fix title using this:
https://www.conventionalcommits.org/en/v1.0.0/

examples:

> refactor(esx_lscustom/client/main.lua): fixed thread
> feat(esx_lscustom/client/main.lua): added new thread for detecting vehicle type
> fix(esx_sit/client/list.lua): fixed mistyped variable
> refactor(esx_legacy): replaced all instances of table.insert with table[#table+1]

mr-capy commented:
> > why failed?
> 
> i guess u can ignore that, but u should fix title using this: https://www.conventionalcommits.org/en/v1.0.0/
> 
> examples:
> 
> > refactor(esx_lscustom/client/main.lua): fixed thread
> > feat(esx_lscustom/client/main.lua): added new thread for detecting vehicle type
> > fix(esx_sit/client/list.lua): fixed mistyped variable
> > refactor(esx_legacy): replaced all instances of table.insert with table[#table+1]

is it good now mine title?

mcNuggets1 commented:
By the way. 
Developers of ESX Legacy, please squash and merge commits, otherwise your git repo looks very unorganised, when merging pull requests.
Didn't see any other form of direct contact, so I'm putting it here.
